addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -434,6 +434,40 @@ def create_employee_from_applicant(self):
         if employee:
             dict_act_window['res_id'] = employee.id
         dict_act_window['view_mode'] = 'form,tree'
+        if len(self) == 1: #only if we create one employee: print rainbow
```

Comment:
This behavior should be extracted in another method. Purpose of this method is to create an employee, not compute stuff for a rainbowman. We could thing of something like having a method action_recruit_employee that calls _create_employee_from_applicant then returns _create_magnificient_rainbowman_with_licorn_head .